core/backend/src/BackendHubAccess.ts

```diff
@@ -188,18 +188,27 @@ export interface CreateNewIModelProps extends IModelNameArg {
  * Generally direct access to these methods should not be required, since higher-level apis are provided.
  * @note This interface is implemented in another repository. Any changes made to this interface must be validated against
  * the implementation found here: https://github.com/iTwin/imodels-clients/blob/main/itwin-platform-access/imodels-access-backend/src/BackendIModelsAccess.ts
- * @internal
+ * @public
  */
 export interface BackendHubAccess {
-  /** Download all the changesets in the specified range. */
+  /**
+   * Download all the changesets in the specified range.
+   * @internal
```

Comment:
@pmconne Should we make `ChangesetFileProps` public instead of being forced to mark the three functions that use it internal?

Reply:
Yes. A package outside of itwinjs-core implements BackendHubAccess, so all of its APIs must be @public and therefore subject to our API support policies.

Reply:
OK, I'll make things public.

Reply:
I assume that means that the other APIs in `BackendHubAccess` that were already explicitly marked as internal also need to be public now?

Reply:
Yes, except the ones marked as deprecated in 3.x - those can be deleted.

Reply:
OK, will do.

Reply:
Because it is so involved, I feel that deleting `downloadV1Checkpoint` should be done in a separate PR.

Reply:
Everything except `downloadV1Checkpoint` is now public. That can be deleted in a separate PR.